mkosi/__init__.py

```diff
@@ -821,12 +802,12 @@ def install_unified_kernel(state: MkosiState, roothash: Optional[str]) -> None:
                 boot_count = f'+{state.root.joinpath("etc/kernel/tries").read_text().strip()}'
 
             if state.config.image_version:
-                boot_binary = state.root / prefix / f"EFI/Linux/{image_id}_{state.config.image_version}{boot_count}.efi"
+                boot_binary = state.root / f"boot/EFI/Linux/{image_id}_{state.config.image_version}{boot_count}.efi"
```

Comment:
Should be maybe go for `efi` as the prefix here? It shouldn't change anything in the UKI case and people who want to do an XBOOTLDR partition would have a mountpoint for it.

Reply:
Not until we drop dracut as it writes to /boot. Maybe in a separate PR